core/install.js

```diff
@@ -14,12 +14,12 @@ var npm = require('npm'),
 	fs = require('fs'),
 	load = deasync(npm.load);
 
-load({loglevel: "silent"});
+load({loglevel: 'silent'});
 var inst = deasync(npm.commands.install),
 upd = deasync(npm.commands.update),
 
 install = function(name) {
-	console.info('Installing "' + name + '" from npm.');
+	console.info('Installing \"' + name + '\" from npm.');
```

Comment:
Inside a single quoted string you shouldn't need to escape a double quote. If that is codacy, tell me the rule and I'll kill it with fire.